src/test/java/BaseTest.java

```diff
@@ -1,11 +1,108 @@
+        driver.manage().window().maximize();
+
+        navigateToPage(baseURL);
+        actions =new Actions(driver);
+    }
+    @AfterMethod
+    public void closeBrowser() {
+        driver.quit();
+    }
+
+
+    public void navigateToPage(String url) {
+        driver.get(url);
+    }
+
+    public void provideEmail(String email) {
+        //WebElement emailField = driver.findElement(By.cssSelector("input[type='email']"));
+        WebElement emailField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[type='email']")));
+        emailField.clear();
+        emailField.sendKeys(email);
+    }
+
+    public void providePassword(String password) {
+        //WebElement passwordField = driver.findElement(By.cssSelector("input[type='password']"));
+        WebElement passwordField =wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[type='password']")));
+        passwordField.clear();
+        passwordField.sendKeys(password);
+    }
+
+    public void clickSubmit() {
+        //WebElement submit = driver.findElement(By.cssSelector("button[type='submit']"));
+        WebElement submit =wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button[type='submit']")));
+        submit.click();
+    }
+    /*public void isAvatarDisplayed() {
+        //WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+        WebElement avatarIcon = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("img[class='avatar']")));
+        Assert.assertTrue(avatarIcon.isDisplayed());
+    }
+
+    public void clickSaveButton() {
+        //WebElement saveButton = driver.findElement(By.cssSelector("button.btn-submit"));
+        WebElement saveButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button.btn-submit")));
+        saveButton.click();
+    }
+
+    public void provideProfileName(String randomName) {
+       // WebElement profileName = driver.findElement(By.cssSelector("[name='name']"));
+        WebElement profileName = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='name']")));
+        profileName.clear();
+        profileName.sendKeys(randomName);
+    }
+
+    public void provideCurrentPassword(String password) {
+        //WebElement currentPassword = driver.findElement(By.cssSelector("[name='current_password']"));
+        WebElement currentPassword = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='current_password']")));
+        currentPassword.clear();
+        currentPassword.sendKeys(password);
+    }
+
+    public String generateRandomName() {
+        return UUID.randomUUID().toString().replace("-", "");
+    }
+
+    public void clickAvatarIcon() {
+        WebElement avatarIcon = driver.findElement(By.cssSelector("img.avatar"));
+        avatarIcon.click();
+    }*/
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.